package.json

```diff
@@ -6,9 +6,14 @@
   "directories": {
     "test": "tests"
   },
+  "typescript": {
+    "definition": "index.d.ts"
+  },
+  "_where": "D:\\dev\\ws\\bizSys\\bizsys-prototype",
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start --single-run",
-    "build": "jspm bundle-sfx src/heracles - jsonld - li - lodash - core-js - core-decorators - rdf-ext - rdf-formats-common - rdf-serializer-jsonld dist/heracles.js --global-name heracles --format cjs --skip-source-maps"
+    "test": "karma start --single-run",
+    "build": "jspm bundle-sfx src/heracles - jsonld - li - lodash - core-js - core-decorators - rdf-ext - rdf-formats-common - rdf-serializer-jsonld dist/heracles.js --global-name heracles --format cjs --skip-source-maps",
+    "buildbig": "jspm bundle-sfx src/heracles + jsonld + li + lodash + core-js + core-decorators + rdf-ext + rdf-formats-common + rdf-serializer-jsonld dist/heracles.js --global-name heracles --format cjs --skip-source-maps"
```

Comment:
yes it was about 40k lines of code, so I dropped that idea :P By the way, when developing, did you rather have the backend or the frontend in mind?